src/components/createTarget/index.js

```diff
@@ -58,11 +60,12 @@ const NewTarget = () => {
 
   const onSubmit = data => {
     if (targetsList.length <= 9) {
-      createTarget({ ...data, ...latLng });
+      createTarget({ ...data, lat, lng });
     } else {
       Swal.fire({ title: t('target.limit.alert'), icon: 'info', confirmButtonColor: '#3085d6' });
```

Comment:
If you are planning on using this `#3085d6` multiple times, I'd consider creating a constant for it. WDYT?

Reply:
Done